frontend/core-ui/src/modules/automations/components/builder/sidebar/components/AutomationNodeLibrarySidebar.tsx

```
@@ -1,91 +1,73 @@
+      >
+        <Tabs.List className="w-full border-b">
+          <Tabs.Trigger value="trigger" className="w-1/2">
+            Triggers
+          </Tabs.Trigger>
+          <Tabs.Trigger value="action" className="w-1/2">
+            Actions
+          </Tabs.Trigger>
+        </Tabs.List>
+
+        {[
+          { type: 'trigger' as 'trigger', list: triggersConst },
```

ESLint

[error] Expected a `const` instead of a literal type assertion.

(@typescript-eslint/prefer-as-const)